client/src/constants/pricing.js

```diff
@@ -326,7 +328,7 @@ export const pricingPackage = [
       'We handle all your marketing needs from strategy to design and implementation.',
   },
   {
-    name: 'Development',
```

Comment:
Really a great job on everything else, but I believe this should just stay as a Development instead of Solutions since this is going to change the name of the link under 'select a package below' section (specifically the word under the camera icon image in the pricing page).